bedrock/firefox/templates/firefox/features/pdf-complete-fr.html

```diff
@@ -0,0 +1,94 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/features/base-article.html" %}
+
+
+{% block article_title_short %}Remplir un PDF en ligne: fiche d’impôts{% endblock %}
+{% block article_title %}Remplir un PDF en ligne : fiche d’impôts{% endblock %}
+{% block page_desc %}Retrouvez un guide pas-à-pas pour remplir et même corriger sa déclaration d’impôts, en utilisant les options intégrées de Firefox.{% endblock %}
+
+{% block article_content %}
+
+<p>Fini les soucis liés à la paperasse, place à la simplicité administrative !  Que vous ayez besoin de remplir une première déclaration, de corriger une erreur ou de gérer vos justificatifs, Firefox vous offre une solution rapide et sécurisée pour vos données. </p>
+
+<p>Remplir des PDF n'a jamais été aussi simple et fiable, même quand on parle de fiches d’impôts.</p>
+
+<h2>Remplir sa PDF</h2>
```

Comment:
**Nit:** Based on copy, this heading should say `Remplir sa déclaration de revenus`?

Reply:
Adria/Leonie approved the copy changes :)